docs/src/examples/components/Popup/Types/PopupNoFocusTrapExample.shorthand.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Button, Input, Popup } from '@stardust-ui/react'
+
+const PopupNoFocusTrapExample = () => (
+  <>
+    <Popup
```

Comment:
Having example here first with focus trap enabled is super confusing.  Yes, it's all correct, but I didn't read carefully and thought it wasn't working.  I would get rid of the 'default' option with it on.

Reply:
will fix, thanks!

Reply:
addressed